package.json

```diff
@@ -13,6 +13,7 @@
     "@react-native-community/masked-view": "^0.1.11",
     "@react-navigation/native": "^5.9.4",
     "@react-navigation/stack": "^5.14.4",
+    "pod-install": "^0.1.22",
```

Comment:
You don't need this line in package.json. When you start working with iOS, you will install [cocoapods](https://cocoapods.org/) for managing IOS dependencies.